test/Interop/Cxx/class/address-only.swift

```diff
@@ -0,0 +1,35 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-build-swift %s -I %S/Inputs -o %t/address_only -Xfrontend -enable-cxx-interop
+// RUN: %target-codesign %t/address_only
+// RUN: %target-run %t/address_only 2&>1
+
+// REQUIRES: executable_test
+
+import TypeClassification
+import StdlibUnittest
+
+var AddressOnlyTestSuite = TestSuite("Address Only Types")
+
+AddressOnlyTestSuite.test("Test struct with copy constructor") {
+  let obj = StructWithCopyConstructorAndValue(value: 42)
+  expectEqual(obj.value, 42)
+}
+
+AddressOnlyTestSuite.test("Test struct with member with copy constructor") {
+  let member = StructWithCopyConstructorAndValue(value: 42)
+  let obj = StructWithMemberWithCopyConstructor(member: member)
+  expectEqual(obj.member.value, 42)
+}
+
+AddressOnlyTestSuite.test(
+  "Test struct with copy constructor and member with copy constructor"
+) {
+  let member = StructWithCopyConstructorAndValue(value: 42)
+  let obj = StructWithCopyConstructorAndMemberWithCopyConstructor(
+    member: member
+  )
+  expectEqual(obj.member.value, 42)
+}
```

Comment:
Could you add the relevant patterns to `type-classification-non-trivial-silgen.swift` & make an IRGen test?

Reply:
Yep. I'll get to this later this week/weekend.

Reply:
Done :)